app/locales/nl.js

```diff
@@ -189,13 +189,13 @@ export default {
   },
   resetPassword: {
     heading: 'Wachtwoord vergeten',
-    newPasswordFormDialog: 'Voer een nieuw wachtwoord in, dan kun je verder met onderzoek doen.',
+    newPasswordFormDialog: 'Voer bijbehorende wachtwoorden in die ten minste 8 tekens lang zijn, zodat u verder kunt gaan met onderzoek.',
```

Comment:
Matching my other suggestion, this would be: "Voer hetzelfde wachtwoord tweemaal in, zodat je verder kunt met onderzoek doen. Een wachtwoord moet tenminste 8 tekens lang zijn."

"Bijbehorende" is the wrong term in any case. That'd translate back as "passwords belonging to", not "matching". Let me know if you need a better literal translation instead of for my own suggestion